src/EmployeeBook.java

```diff
@@ -2,20 +2,28 @@
 
 public class EmployeeBook {
 
-    public static Random RANDOM = new Random();
+    private static final Random RANDOM = new Random();
+
+    public int getRandom(int i, int j) {
+        return RANDOM.nextInt( i, j);
+    }
+
     private final static String [] NAMES = {"Андрей", "Александр", "Иван", "Игорь", "Константин", "Петр", "Семён"};
     private final static String [] SURNAME = {"Андреев", "Иванов", "Петров", "Сидоров", "Синицин", "Лисицин", "Коробов"};
     private final static String [] PATRONYMIC = {"Андреевич", "Александрович", "Иванович", "Игоревич", "Константинович",
             "Петрович", "Семенович"};
 
-    private final static Employee [] EMPLOYEES = new Employee[10];
+    private String employeeFio() {
+        return SURNAME[RANDOM.nextInt(0, SURNAME.length)] + " " +
+                NAMES[RANDOM.nextInt(0, NAMES.length)] + " " +
+                PATRONYMIC[RANDOM.nextInt(0, PATRONYMIC.length)];
+    }
+
+    private final Employee [] EMPLOYEES = new Employee[10];
```

Comment:
в верхнем регистре надо писать только final И static поля. Просто final ИЛИ static надо писать в camelCase
```suggestion
    private final Employee [] employees = new Employee[10];
```